manifests/test_20.0.xml

```diff
@@ -27,7 +27,7 @@
 <project remote="amdlibs" path="openmpapps" name="openmpapps" revision="AOMP-0.5" groups="unlocked" />
 <project remote="tapple"  path="OvO"        name="OvO" revision="master" groups="unlocked" />
 <project remote="qmcpack" path="qmcpack"    name="qmcpack" revision="develop" groups="unlocked" />
-<project remote="sollve"  path="sollve_vv"  name="sollve_vv" revision="master" groups="unlocked" />
+<project remote="sollve"  path="sollve_vv"  name="OpenMP_VV" revision="master" groups="unlocked" />
```

Comment:
I think we should change `path` here  to `OpenMP_VV` as well as this will be the name of the directory it clones into.

And we need to update the `AOMP_SOLVV_REPO_NAME` in `aomp_common_vars`. We can just keep the variable name as is for now so scripts will not have to change:
`-AOMP_SOLVV_REPO_NAME=${AOMP_SOLVV_REPO_NAME:-sollve_vv}`
`+AOMP_SOLVV_REPO_NAME=${AOMP_SOLVV_REPO_NAME:-OpenMP_VV}`

Reply:
Actually run_rocm_test.sh has `sollve_vv` hardcoded. So that will need updated as well.